proto/panacea/datadeal/v0/deal.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+package panacea.datadeal.v0;

```

Comment:
```suggestion
package panacea-oracle.datadeal.v0;
```
Please rename the package and the directory name to avoid conflicts with proto files in the panacea-core repo. https://github.com/medibloc/panacea-core/tree/main/proto/panacea
I'm not sure if we can use a `-` in the package name, but please try.

Reply:
Unfortunately, `-` doesn't work and `_` does.


Now that I think about it, we'd better change it to something like this in the future.
* panacea-core: `pacea.core.datadeal.v2`
* panacea-oracle: `panacea.oracle.datadeal.v0`

Reply:
`panacea.oracle.*` looks reasonable. I don't think we now need to change the package name for panacea-core. It would be enough to change names in this repo to `panacea.oracle.*`.

Reply:
I checked `oracle.proto` in `panacea-core` already exists.
https://github.com/medibloc/panacea-core/blob/main/proto/panacea/oracle/v2/oracle.proto#L2

It seems we need to use an underline (`_`).
`package panacea_oracle.datadeal.v0`

Reply:
Yeah, then using underscore makes sense

Reply:
I changed name to `panacea_oracle`

Reply:
I also made additional changes: https://github.com/medibloc/panacea-oracle/pull/82